test/index.test.ts

```diff
@@ -1,32 +1,32 @@
-import Olympus from "../lib";
+import Olympus from '../lib';
 
-const provider = "https://beta-rpc.mainnet.computecoin.com";
+const provider = 'https://beta-rpc.mainnet.computecoin.com';
```

Comment:
Change json rpc provider to valid network.

Reply:
I have updated the latest test script